README.md

```
@@ -105,6 +105,40 @@ for dt in Quarter(date(1958, 3, 25)):
+
+moreover a week itself can be subscripted, for example:
+
+```
+Week(date(1958, 3, 24))[2]  # date(1958, 3, 26)
+```
+
+one can also slice to get an *generator* that generates `Week`s or `date`s in the week respectively.
+
+The `Week` class itself is also iterable, for example:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)